package/gluon-mesh-vpn-fastd/src/respondd.c

```diff
@@ -229,6 +229,11 @@ static bool get_peer_connection(struct json_object **ret, struct json_object *co
 		struct json_object *jso = json_object_new_double(established_time/1000.0);
 		json_object_set_serializer(jso, json_object_double_to_json_string, "%.3f", NULL);
 		json_object_object_add(*ret, "established", jso);
+
+		if (json_object_object_get_ex(connection, "method", &method)) {
+			const char* method_value = json_object_get_string(method);
+			json_object_object_add(*ret, "method", method_value);
```

Comment:
This doesn't look right - I guess we should add some `-Werror` flags to make the build fail for implicit pointer casts (from `const char*` to `struct json_object*` in this case).  You need a `json_object_get()` however to adjust the reference count.

```suggestion
			json_object_object_add(*ret, "method", json_object_get(method));
```

Reply:
Applied, thanks.